### DISCUSSION TITLE
Criar fórum Back-end Brasil

### DISCUSSION THREAD

**Discussion by rhuangabrielsantos** [frontendbr/forum#2016] [Discussão]
Olá amigos!

Gostaria de compartilhar com vocês uma ideia que tive recentemente (ontem pra ser mais exato haha), que é criar um repositório semelhante a este só que voltado ao mundo do back-end. 

Pesquisando pelo github eu encontrei uma organização chamada [Back-end Brasil](https://github.com/backend-br), só que ao contrário desta comidade que estamos, não sinto que é ativa e que existe ajuda entre os membros. Qual a opnião de vocês sobre isso?

Se quiserem passar por lá para contruirmos juntos esta comunidade, todos serão muito bem vindos.

Link: https://github.com/backendbr/forum

Ps: Eu me baseei muito neste repositório, se tiver algum problema por favor me avisem que apago o repositório.

**Comment by wilsonneto-swe** [comment]
Acho que seria mais legal se juntar a galera que já está tentando fazer um fórum o backend do que criar um novo, a força da comunidade é unir, troca uma ideia com os caras e veja se pode se juntar a eles.
Mas bom, boa sorte ai, abraço!

**Comment by victorferreira** [comment]
Pra mim a grande dificuldade em reunir uma comunidade de backend é conseguir agregar a galera de várias linguagens. Uma das grandes vantagens do front é que a stack de tecnologia é padrão e não dá pra fugir de HTML + CSS + JS.

**Reply by felipefialho** [reply]
Mano isso faz bastante sentido viu...

**Reply by victorferreira** [reply]
O fato da stack ser pré-definida remove qualquer discussão de qual linguagem é melhor pra resolver um determinado problema. Embora a gente tenha movido a discussão sobre qual framework é melhor (é React, claro! 😜), é muito mais fácil navegar na comunidade já que a base é compartilhada.

**Reply by ninetails** [reply]
Mas ainda sim, apesar da linguagem ser variada, existe uma coisa em comum que é **ser voltado a web**. Hoje, falando de microsserviços, muito se fala em usar a melhor stack para determinado fim, mas existem coisas em comum como comunicação e filas por exemplo.

Aliás... aplicações inclusive podem ser abstraídas em _containeirização_, nem importa em qual linguagem foi feita rs.

Mas aí concordo que é uma conversa muito além e que no geral ia ter mais perguntas sobre PHP, talvez Python e Ruby, muito menos provável sobre DBs...
